packages/bcer-retailer-app/app/src/components/Noi/NoiTable.tsx

```diff
@@ -7,11 +7,12 @@ import { StyledTable } from 'vaping-regulation-shared-components';
 interface NoiTableProps {
   readonly data: ReadonlyArray<BusinessLocation>;
   type?: string;
+  downloadAction?: Function;
   [s: string]: unknown;
 }
 
 function NoiTable({ data, type, ...props }: NoiTableProps): JSX.Element {
-  let columns=[
+  let columns :any = [
```

Comment:
do we know the typing of this? if so we should probably create a typedef for it, or at the least in-line type it